src/storage/memory.ts

```diff
@@ -0,0 +1,14 @@
+import { UmzugStorage } from './contract';
+
+export const memoryStorage = (): UmzugStorage => {
+	let executed = [] as string[];
+	return {
+		logMigration: async name => {
+			executed.push(name);
+		},
+		unlogMigration: async name => {
+			executed = executed.filter(n => n !== name);
+		},
```

Comment:
I agree, but this is mirroring the implementation of JSONStorage. Let's make both throw before releasing v3. I think it should go in as a standalone PR though so it can be marked as a separate breaking change.